src/h5web/toolbar/LineToolbar.tsx

```diff
@@ -27,7 +27,9 @@ function LineToolbar(): ReactElement {
         role="radiogroup"
         ariaLabel="Curve type"
         value={curveType}
-        onChange={setCurveType}
+        onChange={(val) => {
+          setCurveType(val as CurveType);
+        }}
```

Comment:
Type error (`val` is `string` and it's too difficult to type the `ToggleGroup` context with a generic). Not sure why it hadn't been detected before.